blog.json

```diff
@@ -1,9 +1,19 @@
 [
   {
+    "uuid": "126aa8c4-9f08-4813-a05b-ad0a8d18f60b",
+    "slug": "introducing-the-exercism-blog",
+    "category": "updates",
+    "published_at": "2019-01-16 12:00",
+    "content_repository": "blog",
+    "content_filepath": "posts/introducing-the-exercism-blog.md",
+    "title": "Introducing the Exercism blog",
+    "author_handle": "iHiD",
+    "marketing_copy": "Each day Exercism gets hundreds of new users, thousands of new submissions, and dozens of improvements across its language tracks and product, but many people have no idea how much is going on or how active the site is. Today we're launching our new blog in order to change that!"
+  }, {
```

Comment:
Maybe want a newline after `,`?

Reply:
The mentor .json files have a newline in instances like this as do the tracks' config.json file.  I only pointed it out because it stuck out to me.

Reply:
OK. Great. I've changed it. Thanks :)